documentation/index.rst

```diff
@@ -26,6 +26,16 @@ check:
    content/developmentworkflow
    content/tracingandprofiling
 
+Kw Handbook
```

Comment:
We should make it clear to readers that this handbook is directed to contributors, so make it something like `Kw Contributor's Handbook` (feel free to propose other possibilities). We need to nail the name here as this may be decisive on converting newcomers as contributors.